.github/workflows/build-android-app.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       with:
         node-version: 16
         cache: 'npm'
-
+    - run: npm i -g @mapbox/node-pre-gyp
```

Comment:
What is the purpose of this build step?  Is there some browser dependency that needs node-gyp or is this leftover from when you were using the older webtorrent tracker library?